src/Handler/Locator/ContainerLocator.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace League\Tactician\Handler\Locator;
+
+use Interop\Container\ContainerInterface;
+use League\Tactician\Exception\InvalidCommandException;
+use League\Tactician\Exception\MissingHandlerException;
+use League\Tactician\Handler\CommandNameExtractor\CommandHandlerNameExtractor;
+
+/**
+ * Fetch handler instances from a Dependency Injection Container supporting the Interop specification.
+ *
+ * This locator allows you to bind a handler object from a container to receive commands
+ * of a certain class name. For example:
+ *
+ *      // Instantiate locator
+ *      $containerLocator = new ContainerLocator($container);
+ *
+ *      // Returns an instance of 'My\TaskAddedHandler' from the container.
+ *      $containerLocator->getHandlerForCommand('My\TaskAddedCommand');
+ *
+ * The default behaviour for this locator is to take the Command's Fully Qualified Class Name, suffix it with
+ * 'Handler' and ask the container if a class with that name can be instantiated.
+ *
+ * Sometimes you want to specify the name of the handler yourself or you container uses identifiers to retrieve objects,
+ * in this case it is possible to provide a second argument to the constructor, or use the addHandler method, to
+ * provide the name with which the handler will be located in the Dependency Injection Container.
+ *
+ * For example:
+ *
+ *      // Instantiate locator using mapping array
+ *      $containerLocator = new ContainerLocator($container, [ 'My\TaskAddedCommand' => 'my.handler.identifier' ]);
+ *
+ *      // Or by using the addHandler method
+ *      $containerLocator->addHandler('My\TaskRemovedCommand', 'my.handler.removed.identifier');
+ *
+ *      // Returns an the instance belonging to the key 'my.handler.identifier' from the container.
+ *      $containerLocator->getHandlerForCommand('My\TaskAddedCommand');
+ */
+final class ContainerLocator implements HandlerLocator
+{
+    /** @var ContainerInterface */
+    private $container;
```

Comment:
@mvriel 

For consistency, should we turn this into a multi-line docblock, rather than single line comment

```php
/**
 * @var ContainerInterface
 */
```

?